package.json

```diff
@@ -10,18 +10,27 @@
     "format": "npx prettier --write ."
   },
   "dependencies": {
+    "@radix-ui/react-avatar": "^1.0.3",
```

Comment:
isnt there a way to install only radix-ui?

Reply:
each primitives are individual and versioned independently, you just pick what you want. They are tiny modules.

Reply:
ok